packages/shared/src/components/widgets/SocialShare.tsx

```diff
@@ -25,15 +25,27 @@ import { Comment, getCommentHash } from '../../graphql/comments';
 import { useAuthContext } from '../../contexts/AuthContext';
 import { useLazyModal } from '../../hooks/useLazyModal';
 import { LazyModal } from '../modals/common/types';
-import { SourcePermissions, Squad } from '../../graphql/sources';
+import { SourcePermissions, Squad as SquadType } from '../../graphql/sources';
```

Comment:
What was the conflict here, the inline SocialShareType?
If so it might be best to move that to a lib file instead.

Reply:
This was the conflict, i'll move it to lib